tasks/system.yml

```diff
@@ -14,7 +14,7 @@
     undetermined_actions:
       - "{{default_notification.notification_method_id}}"
   with_items:
-    - { name: "Host Status", description: "Alarms when the specified host is down or not reachable", severity: "HIGH", expression: "host_alive_status > 0", match_by: ["target_host", "hostname"]}
```

Comment:
I agree I could go either way on this. Dexter was working a case where the intent was to add more information to an alarm notification and have it be one alarm. That way there were fewer alarms to deal with but when it triggered you had the info to know if it was just one observer or multiple showing the issue. Without this change I would lean toward more alarms so you could more easily figure out if it was a particular network issue.